src/gstools/field/generator.py

```diff
@@ -532,3 +538,332 @@ def _create_unit_vector(self, broadcast_shape, axis=0):
         e1 = np.zeros(shape)
         e1[axis] = 1.0
         return e1
+
+
+class Fourier(Generator):
+    r"""Fourier method for calculating periodic, isotropic random fields.
+
+    Parameters
+    ----------
+    model : :any:`CovModel`
+        Covariance model
+    mode_no : :class:`list`
+        Number of Fourier modes per dimension.
+    period : :class:`list`, optional
+        The spatial periodicity of the field, is often the domain size.
+    seed : :class:`int`, optional
+        The seed of the random number generator.
+        If "None", a random seed is used. Default: :any:`None`
+
+    **kwargs
+        Placeholder for keyword-args
+
+    Notes
+    -----
+    The Fourier method is used to generate periodic isotropic spatial random
+    fields characterized by a given covariance model.
+    The calculation looks like:
+
+    .. math::
+       u\left(x\right)=
+       \sum_{i=1}^{N}\sqrt{2S(k_{i})\Delta k}\left(
+       Z_{1,i}\cdot\cos\left(\left\langle k_{i},x\right\rangle \right)+
+       Z_{2,i}\cdot\sin\left(\left\langle k_{i},x\right\rangle \right)
+       \right)
+
+    where:
+
+        * :math:`S` : spectrum of the covariance model
+        * :math:`Z_{j,i}` : random samples from a normal distribution
+        * :math:`k_i` : the equidistant Fourier grid
+    """
+
+    def __init__(
+        self,
+        model,
+        mode_no,
```

Comment:
Could we provide a default value for `mode_no` like with the RandMeth class?
Maybe something like `model.dim * [32]` following your examples?

Reply:
And I would like some syntactic sugar, to be able to only give one number that is then used for all dimensions depending on the cov-model.

Reply:
Then we could use `mode_no=32` as default ;-)

Reply:
good idea, implemented.